homeassistant/components/litejet/config_flow.py

```diff
@@ -59,15 +59,12 @@ async def async_step_user(
         if user_input is not None:
             port = user_input[CONF_PORT]
 
-            await self.async_set_unique_id(port)
-            self._abort_if_unique_id_configured()
```

Comment:
Any reason why this is not unique anymore?

Reply:
The port value isn't considered stable under HA's rules so it shouldn't have been being used here. Unfortunately the LiteJet system doesn't provide any identity information over its protocol so there's nothing else to use.

This was in response to MartinHjelmare post-review comment in my last PR for this component: https://github.com/home-assistant/core/pull/44409#discussion_r582469728

Reply:
Right, but shouldn't it be prevented it being set up for the same port twice?

Reply:
I'd expect that to be guarded by the OS not allowing the serial port to be opened twice. Guarding that by path is treacherous anyway since there is at least one symbolic link to the serial port device on Linux. Or if using pyserial's TCP features there are multiple IP or hostname equivalent paths. It'd be a very minor best effort protection.

This scenario is currently blocked by the single instance requirement earlier in the config flow. Your line of thinking is the reason I originally put something in there which I figured was better than nothing in case that requirement was ever removed.

This isn't something I am passionate about either way; it happened to be open in my git since that last PR and got swept up into this PR without much thought. I'm happy to undo the change to those two lines.

Reply:
> This scenario is currently blocked by the single instance requirement earlier in the config flow.

That I missed/wasn't visible in the diff view! That resolved my concern 👍 

Thanks for the clarification.